eth_account/account.py

```diff
@@ -731,16 +733,56 @@ def sign_transaction(self, transaction_dict, private_key, blobs=None):
-             's': 37050226636175381535892585331727388340134760347943439553552848647212419749796,
-             'v': 0}
+            SignedTransaction(
+                rawTransaction=HexBytes('0x01f8ad82053922843b9aca00830186a09409616c3d61b3331fc4109a9e41a8bdb7d9776609865af3107a400086616...'),
+                hash=HexBytes('0xca9af2ef41691e06eb07e02125938fd9bb5a311e8daf330b264e77d6cdf3d17e'),
+                r=107355854401379915513092408112372039746594668141865279802319959599514133709188,
+                s=6729502936685237038651223791038758905953302464070244934323623239104475448298,
+                v=1
+            )
             >>> w3.eth.sendRawTransaction(signed.rawTransaction)
+
+
+        .. code-block:: python
+
+            >>> blob_transaction = {
+                    "type": 3,  # optional - can be implicitly determined based on `maxFeePerBlobGas` param
+                    "gas": 100000,
+                    "maxFeePerGas": 2000000000,
+                    "maxPriorityFeePerGas": 2000000000,
+                    "maxFeePerBlobGas": 2000000000,
+                    "data": "0x616263646566",
+                    "nonce": 34,
+                    "to": "0x09616C3d61b3331fc4109a9E41a8BDB7d9776609",
+                    "value": "0x5af3107a4000",
+                    "accessList": (  # optional
+                        {
+                            "address": "0x0000000000000000000000000000000000000001",
+                            "storageKeys": (
+                                "0x0100000000000000000000000000000000000000000000000000000000000000",
+                            )
+                        },
+                    ),
+                    "chainId": 1337,
+
+                    # Note: unless you've computed `blobVersionedHashes` yourself and it matches the blob data you are
+                    # sending along with the tx, you should omit the  field since it will be calculated from the blobs
+                    # "blobVersionedHashes": [...]  # optional (but must match the expected value from the blob data)
+                }
+            >>> empty_blob = b"\x00" * 32 * 4096  # 4096 empty 32-byte field elements
+            >>> key = '0x4c0883a69102937d6231471b5dbb6204fe5129617082792ae468d01a3f362318'
+            >>> signed = Account.sign_transaction(blob_transaction, key, blobs=[empty_blob])
+            SignedTransaction(
+                rawTransaction=HexBytes('0x03fa020147f8d98205392284773594008477359400830186a09409616c3d61b3331fc4109a9e41a8bdb7d97766098...'),
+                hash=HexBytes('0xf9dc8867c4324fd7f4506622aa700989562770f01d7d681cef74a1a1deb9fea9'),
+                r=14319949980593194209648175507603206696573324965145502821772573913457715875718,
+                s=9129184742597516615341309773045281461399831333162885393648678700392065987233,
+                v=1
+            )
         """  # noqa: E501
```

Comment:
nit: 
```suggestion
        """
```

Reply:
I think this is necessary at the end of the docstring so that all lines in the docstring that break `E501` are accounted for. Unless I missed something but we can probably address it elsewhere if you found a way around that.